tests/gocase/unit/applybatch/applybatch_test.go

```diff
@@ -0,0 +1,39 @@
+package applybatch
+
+import (
+	"context"
+	"encoding/hex"
+	"testing"
+
+	"github.com/apache/kvrocks/tests/gocase/util"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApplyBatch_Basic(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{})
+	defer srv.Close()
+
+	ctx := context.Background()
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+
+	t.Run("Make sure the apply batch command works", func(t *testing.T) {
+		// SET a 1
+		batch, err := hex.DecodeString("04000000000000000100000003013105010D0B5F5F6E616D6573706163656106010000000031")
```

Comment:
Is it obtained by dumping the WriteBatch?

Reply:
Yes, I printed the batch in https://github.com/apache/kvrocks/blob/unstable/src/cluster/replication.cc#L123